src/helpers.test.ts

```diff
@@ -107,16 +107,16 @@ describe('calcDiscrepancy:', () => {
 
 })
 
-//Es necesario obtener el prefix de acuerdo a la zona donde se encuentra el usuario.
 describe('currencyFormatter:', () => {
-  // Mock de formatNumber para pruebas
+  // Solo funciona con prefix = 'USD', no funciona con '€' ni con '$'
```

Comment:
Colocar comentarios en ingles

Reply:
fixed, comments corrected, now they are in spanish.